EFSAdvent/data/actorinfo/BCK2.txt

```diff
@@ -1,4 +1,4 @@
-BCK2 (color block for 1 person)
+BCK2 (color block for 1 person) Has GBA sprite
```

Comment:
It may be better and more efficient to only mention if something is not working?